### DISCUSSION TITLE
Using Robyn on subnational data

### DISCUSSION THREAD

**Discussion by hugmal** [facebookexperimental/Robyn#1036] [General]
Hi,
I understand MMM models are most commonly applied to data at the national level, with one measurement per day or week only.
Is there a reason why one cannot/should not apply Robyn to data at the regional level (e.g. 16 daily measurements for each of 16 regions).  Would a geosplit cause problems with the algorithm?; I.e. in some of our regions we do not show adverts in some of the channels?

**Comment by AnantChandra** [comment]
Regional-level Market Mix Modeling with Robyn isn't just possible—it's often advantageous. 

Using your 16 daily measurements across 16 regions provides more data points for robust statistical analysis. Also, the geo-splits create natural experiments that help with attribution. You can either build separate models per region or create one unified model with regional factors and interaction terms. 

The key is accounting for region-specific variables like local seasonality patterns, demographic differences, and competitive landscapes. This reveals geographic differences in marketing effectiveness that national-level modeling would miss entirely.
